molecule/resources/prepare.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Prepare
+  hosts: all
+  gather_facts: true
+  tasks:
+    - name: Install ansible support packages
+      package:
+        name: "{{ item }}"
+        state: present
+      with_items:
+        - ca-certificates
```

Comment:
If this package is required for minio to work on Debian, it should be in role not in test suite.

Reply:
It is only required when running in a Docker container.